src/com/trilead/ssh2/signature/KeyAlgorithmManager.java

```diff
@@ -38,10 +38,7 @@ private KeyAlgorithmManager() {
             // we don't use ECDSA algorithms in this case
         }
 
-        // https://tools.ietf.org/html/rfc8332
-        algorithms.add(new RSAKeyAlgorithm("SHA256withRSA", "rsa-sha2-256"));
```

Comment:
this might affect my comment: 
https://github.com/jenkinsci/trilead-ssh2/pull/60#issuecomment-735609208

Reply:
we recently added support for them https://github.com/jenkinsci/trilead-ssh2/pull/47